src/main/java/webserver/data/JsonClient.java

```diff
@@ -75,4 +75,21 @@ public void updateItemById(int id, JSONObject updatedItem) {
             e.printStackTrace();
         }
     }
+
+    public void deleteItemById(int id) {
+        JSONArray todos = getAllItems();
+        try (FileWriter file = fileUtils.fileWriter(todoFilePath)) {
+            for (int i = 0; i < todos.size(); i++) {
+                JSONObject obj = (JSONObject) todos.get(i);
+                int idToMatch = Integer.parseInt(obj.get("id").toString());
+                if (id == idToMatch) {
+                    todos.remove(i);
+                }
+            }
+            file.write(todos.toJSONString()); 
+            file.flush();
```

Comment:
Doesn't hurt, but you probably don't need to flush the stream.  This will be done automatically when you close it (which happens with you exit the try-with-resource block).